src/shared/components/paginationControls/PaginationControls.tsx

```diff
@@ -44,6 +46,7 @@ export class PaginationControls extends React.Component<IPaginationControlsProps
         itemsPerPage: SHOW_ALL_PAGE_SIZE,
         itemsPerPageOptions: [10, 25, 50, 100],
         showAllOption: true,
+        textBeforeButtons:"",
         textBetweenButtons: "text btwn buttons",
```

Comment:
the show more button text can't be changed atm. So only if `showMoreButton==false`, you have `textBetweenButtons`, so the name still makes sense. We could update it to `textForMiddleButton` and use that to set the `show more` text as well, but it's not really necessary atm imo, since the text is always the same